awscli/examples/macie2/enable-macie.rst

```diff
@@ -0,0 +1,25 @@
+**To enable Amazon Macie**
+
+The following ``enable-macie`` example enables Amazon Macie for the current AWS account and Region. ::
+
+    aws macie2 enable-macie
+
+Output::
+
+    {
+        "status": "ENABLED"
+    }
+
```

Comment:
Please provide a "For more information" link for every example in this request. See format in the contribution guide: [aws.github.io/aws-cli/docs_styleguide.html#for-more-information-link](https://aws.github.io/aws-cli/docs_styleguide.html#for-more-information-link)